src/Core/Framework/Changelog/ChangelogDefinition.php

```diff
@@ -20,8 +20,6 @@ class ChangelogDefinition
     #[Assert\NotBlank(message: 'The title should not be blank')]
     private string $title;
 
-    #[Assert\NotBlank(message: 'No issue was referenced')]
-    #[Assert\Regex(pattern: '/^(NEXT-|#)\d+$/', message: 'The issue has an invalid format')]
```

Comment:
Does this mean there is no validation at all for this field? Shouldn't we still insure that if it contains an `issue`, that it at least is a syntactical correct reference to a GH / Jira Issue? 🤔

Reply:
IMHO, that field should not be used often anymore, because the default is to use the fixes #xxx from the commit or it will just use the PR number.

If we add them again I have to look up the PR in another way, because the reference is added to the title on merge.